src/webgpu/api/validation/texture/destroy.spec.ts

```diff
@@ -3,17 +3,61 @@ Destroying a texture more than once is allowed.
 `;
 
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUConst } from '../../../constants.js';
 import { ValidationTest } from '../validation_test.js';
 
+const descriptor: GPUTextureDescriptor = {
+  size: [1, 1, 1],
+  format: 'rgba8unorm',
+  usage: GPUConst.TextureUsage.RENDER_ATTACHMENT | GPUConst.TextureUsage.SAMPLED,
+};
+
 export const g = makeTestGroup(ValidationTest);
 
-g.test('twice').fn(async t => {
-  const tex = t.device.createTexture({
-    size: [1, 1, 1],
-    format: 'r8unorm',
-    usage: GPUTextureUsage.SAMPLED,
-  });
+g.test('it_is_valid_to_destroy_a_texture').fn(t => {
+  const texture = t.device.createTexture(descriptor);
+  texture.destroy();
+});
 
-  tex.destroy();
-  tex.destroy();
+g.test('it_is_valid_to_destroy_a_destroyed_texture').fn(t => {
+  const texture = t.device.createTexture(descriptor);
+  texture.destroy();
+  texture.destroy();
 });
+
+g.test('it_is_invalid_to_submit_a_destroyed_texture_before_and_after_encode')
```

Comment:
To conform to newer style, please make these test names shorter and put the detail in `.desc()` instead. Every test should have a `.desc()` and we should update them as opportunistically as we can, even if the descriptions are shortish, e.g. "It is invalid to submit with a texture that is destroyed {before, after} encoding finishes"

Reply:
Done.